langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiChatModelName.java

```diff
@@ -0,0 +1,42 @@
+package dev.langchain4j.model.azure;
+
+/**
+ * You can get the latest model names from the Azure OpenAI documentation or by executing the Azure CLI command:
+ * az cognitiveservices account list-models --resource-group "$RESOURCE_GROUP" --name "$AI_SERVICE" -o table
+ */
+public enum AzureOpenAiChatModelName {
+
+    GPT_3_5_TURBO("gpt-3.5-turbo"), // alias for the latest gpt-3.5-turbo model
```

Comment:
Shouldn't it (and the rest of 3.5 models here and in `AzureOpenAiLanguageModelName`) be `gpt-35-turbo`? AFAIK Azure deployment names do not have dots.

One way to test it would be to iterate over all enums with `maxTokens=1`, like `AnthropicChatModelIT.should_support_all_enum_model_names()`

Reply:
Yes, you're right. Dot `.` are not allowed. 

I've added a few tests so I can test all the model names. But due to quota restriction, it was impossible to test them all at once. I had to test bunch by bunch... but all the tests pass.

I've also updated the shell script so it creates all the models.

Let me know what you think